www/src/data/sidebars/doc-links.yaml

```diff
@@ -109,16 +109,16 @@
                 # This list should only include the top 5 CMSs by Gatsby plugin usage.
                 # Please do not add your headless CMS here.
                 # Instead, please add links on the /docs/headless-cms/ page
-                - title: Sourcing from WordPress
-                  link: /docs/sourcing-from-wordpress/
-                - title: Sourcing from Drupal
-                  link: /docs/sourcing-from-drupal/
                 - title: Sourcing from Contentful
                   link: /docs/sourcing-from-contentful/
-                - title: Sourcing from Prismic
-                  link: /docs/sourcing-from-prismic/
                 - title: Sourcing from Netlify CMS
                   link: /docs/sourcing-from-netlify-cms/
+                - title: Sourcing from WordPress
```

Comment:
What does the change in order represent? It doesn't look alphabetical...

Reply:
Like the [comments](https://github.com/gatsbyjs/gatsby/blob/5bb6adec3b63d533aa570c48f485b4cd614c4971/www/src/data/sidebars/doc-links.yaml#L109-L111) above it say there should only be the top 5 CMSs and I reordered them based off of their rank in the plugin library (ie how many downloads they get, which maybe isn't the _best_ metric)

Drupal was an exception, it's the 5th one even though it's used a little less it seems to be a focus that it's supported.

Reply:
Maybe it would be worth adding more than 5 though?

Reply:
This makes sense, I was just curious! I think we should keep it at 5. Drupal is a strategic choice, so that's why it's there.